### PR TITLE
io.github.libvibrant.vibrantLinux

### PR DESCRIPTION
**Pull request by Scrumplex**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/libvibrant/vibrantLinux
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build io.github.zeemzha.vibrantLinux

File: 0002-refactor-packaging-install-to-app.patch [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+diff --git a/vibrantLinux.pro b/vibrantLinux.pro
+index 9792f94..78c0e2c 100644
+--- a/vibrantLinux.pro
++++ b/vibrantLinux.pro
+@@ -59,7 +59,7 @@ FORMS += \
+ 
+ # Default rules for deployment.
+ qnx: target.path = /tmp/$${TARGET}/bin
+-else: unix:!android: target.path = /usr/bin
++else: unix:!android: target.path = /app/bin
```
[the remark this answers is not in the document]
Comment (Scrumplex) [reply 0002-refactor-packaging-install-to-app.patch:19]: It seems like flatpak-builder doesn't call qmake with any defined `PREFIX`

[the remark this answers is not in the document]
Comment (hfiguiere) [reply 0002-refactor-packaging-install-to-app.patch:19]: the code here show it wouldn't matter. It's not at all a problem here for the flatpak.

**Comment by hfiguiere** [comment]
bot, build io.github.zeemzha.vibrantLinux

File: 0002-refactor-packaging-install-to-app.patch (hunk 1 above)
[the remark this answers is not in the document]
Comment (Scrumplex) [reply 0002-refactor-packaging-install-to-app.patch]: qmake apparently doesn't have any standard way of doing this :|

Apparently flatpak-builder doesn't even know what to do here: https://github.com/flatpak/flatpak-builder/blob/60c11d081cfa06742f4f9ea6f0fc1ccd13c5b2ca/src/builder-module.c#L1784

**Comment by Scrumplex** [comment]
We are currently planning to move the source repository somewhere else, as the original maintainer doesn't have interest to maintain the software any further. I will be making the appropriate changes so that we can drop the patches included in this PR after tagging a new release.

**Comment by hfiguiere** [comment]
bot, build io.github.zeemzha.vibrantLinux

**Comment by hfiguiere** [comment]
my bad

bot, build io.github.libvibrant.vibrantLinux

**Comment by Scrumplex** [comment]
Sounds almost like I have a typo somewhere. Let me check

**Comment by Scrumplex** [comment]
Found the culprit. Let me fix that

**Comment by Scrumplex** [comment]
There we go.

**Comment by hfiguiere** [comment]
bot, build io.github.libvibrant.vibrantLinux

**Comment by hfiguiere** [comment]
(btw you can also trigger the build)

**Comment by Scrumplex** [comment]
> (btw you can also trigger the build)

oh, well :D

So there isn't anything new from my side here. So this should be fine

**Pull request closed without merging**